server/chat-plugins/scavengers.ts

```diff
@@ -1686,21 +1972,25 @@ const commands = {
 
 		if (!sortingFields.includes(sortMethod)) sortMethod = 'finish'; // default sort method
 
-		PlayerLeaderboard.visualize(sortMethod).then(raw => {
+		PlayerLeaderboard.visualize(sortMethod).then(data => {
```

Comment:
This should also be refactored into an async function. Can you do that for me? I can guide you through it if you have trouble.